lib/bungee/src/Input.cpp

```
@@ -0,0 +1,65 @@
+#include <numbers>
+
+namespace Bungee {
+
+using namespace Internal;
+
+namespace {
+static constexpr float pi = std::numbers::pi_v<float>;
+static constexpr float gain = (3 * pi) / (3 * pi + 8);
+} // namespace
+
+Input::Input(int log2SynthesisHop, int channelCount, Fourier::Transforms &transforms) :
```

Cppcheck (2.19.0)

[information] Include file

(missingIncludeSystem)